csv/src/test/java/org/apache/metamodel/csv/CsvDataContextTest.java

```diff
@@ -492,7 +490,7 @@ public void testMaterializeTable() throws Exception {
     public void testAlternativeDelimitors() throws Exception {
         File file = new File("src/test/resources/csv_semicolon_singlequote.csv");
         CsvDataContext dc = new CsvDataContext(file, semicolonConfiguration);
-        Table table = dc.getSchemas().get(0).getTables().get(0);
```

Comment:
This tripped me up for a while, it turns out _getSchemas().get(0)_ returns the schema for information_schema while _getDefaultSchema()_ returns the schema for specified resource. Now that the typing on the schema matters for reads dataSet.next() includes a cast which was failing.